src/box/lua/console.c

```diff
@@ -421,18 +421,19 @@ luaT_console_setup_write_cb(struct lua_State *L)
 	if (!console_hide_show_prompt_is_enabled())
 		return;
 
-	say_set_stderr_callback(console_hide_prompt, console_show_prompt);
-
-	lua_getfield(L, LUA_GLOBALSINDEX, "package");
-	lua_getfield(L, -1, "loaded");
-	lua_getfield(L, -1, "internal.print");
+	lua_getfield(L, LUA_GLOBALSINDEX, "require");
+	lua_pushstring(L, "internal.print");
+	if (luaT_call(L, 1, 1) != 0)
```

Comment:
Regarding `LUA_OK`: I don't agree with @Buristan, since `luaT_call` yields `0` on success and `1` on failure.

Reply:
I agree with myself (and Igor) here: we use just 0 in tarantool for checking `luaT_call()` and `lua_pcall()` return values, so I'll keep it consistent. Moreover, technically `lua_pcall()` is described as a function that returns zero at success (not `LUA_OK` constant). And, yes, it is `luaT_call()`, which has no good description, but its source code has `return 0;` at the end.

Reply:
OK, my bad.